docs/changes/0.5.0.rst

```diff
@@ -0,0 +1,52 @@
+Changelog for QCoDeS 0.5.0
+==========================
+
+The July 2019 release of QCoDeS
+
+## New
+- `ElapsedTimeParameter` (#1629)
+- `load_by_run_spec` enables to load data set given run specifications, including the `run_id` (#1637)
+- Linking runs (#1639)
+- allow parameters to be excluded from snapshot (#1653)
+
+## Improved
+- improved docs
+  - JSON serializability of snapshot (#1627)
+  - Tektronix AWG 7000/5028 resolution docstring (#1628)
+  - m4i docstrings (#1638)
+  - Configuring QCoDeS, Station notebooks (#1630) 
+- installed QCoDeS package marked as safe for type checking (#1634)
+- log installation info (#1648)
+- allow for using application insight for telemetry (#1648)
+- fix bug in `on_off_mapping` (#1655)
```

Comment:
This should be in drivers: `fix incorrect use of on_off_mapping in Keysight E8267D driver`